### DISCUSSION TITLE
When is the next release planned ?

### DISCUSSION THREAD

**Discussion by eddycharly** [open-telemetry/opentelemetry-go#3502] [General]
When is the next release planned ?

It looks like prometheus exporter is not usable in the last release, can we hope for another release soon ?

**Comment by MrAlias** [comment]
Please see the current milestones for work that needs to be completed for the releases:

- https://github.com/open-telemetry/opentelemetry-go/milestone/34
- https://github.com/open-telemetry/opentelemetry-go/milestone/33

**Reply by eddycharly** [reply]
Thanks, this doesn't help me guessing a potential date though.

**Reply by ktong** [reply]
I am also waiting for a new release, is there any update on due date?
